app/code/scout/ScrapePage.server.ts

```diff
@@ -122,57 +125,31 @@ async function scrapePageImpl(urlStr: string): Promise<ScrapedInfo> {
 
     const twitterUsername = await twitter.getTwitterHandle(root);
 
-    if (twitterUsername && TWITTER_BEARER_TOKEN.length > 0) {
-      const options = {
-        headers: {
-          Authorization: `Bearer ${TWITTER_BEARER_TOKEN}`
-        }
-      };
-      const getUserEndpoint =
-        "https://api.twitter.com/2/users/by?usernames=" + twitterUsername;
-      const getUserResponse1 = await fetch(getUserEndpoint, options);
-      const getUserData = await getUserResponse1.json();
-      const userId = getUserData.data[0].id;
-      const getTweetsEndpoint = `https://api.twitter.com/2/users/${userId}/tweets?max_results=25`;
-      console.log("\n= = = = = SENDING TWITTER REQUEST = = = = = \n");
-      const getTweetsResponse = await fetch(getTweetsEndpoint, options);
-      const getTweetsData = await getTweetsResponse.json();
-      const lastTweet = getTweetsData.data[getTweetsData.data.length - 1];
-      console.log("response from twitter:\n" + JSON.stringify(getTweetsData));
-      console.log("LAST TWEET: ", lastTweet);
-      twitter.getUserData(twitterUsername).then((data) => {
-        const user = data.data[0];
-        const authorName = user.name;
-        const description = user.description;
-        const profileImageUrl = user.profile_image_url;
-        const followersCount = user.public_metrics.followers_count;
-        console.log("Author name:", authorName);
-        console.log("Description:", description);
-        console.log("Profile image URL:", profileImageUrl);
-        console.log("Followers:", followersCount);
-
-        return {
-          url: url,
-          fullUrl: url,
-          hash,
-          title,
-          contentType: "twitter",
-          summary: description,
-          authorName: authorName,
-          image: profileImageUrl,
-          likes: followersCount
-        };
-      });
+    if (twitterUsername) {
+      let twitterObject;
+      try {
+        twitterObject = await twitter.fetchTwitterData(twitterUsername);
+      } catch (error) {
+        console.error("Failed to fetch Twitter data: ", error);
+      }
+      console.log("TWITTER OBJECT: ", twitterObject);
+      return twitterObject;
```

Comment:
let's talk about this on Wednesday.  To think about:  what is the job / responsibility of this scrapePage function?  How does that relate to the info from twitter?